front/src/modules/ui/tab/components/Tab.tsx

```diff
@@ -21,12 +21,19 @@ const StyledTab = styled.div<{ active?: boolean }>`
   display: flex;
   gap: ${({ theme }) => theme.spacing(1)};
   justify-content: center;
-  padding: ${({ theme }) => theme.spacing(2) + ' ' + theme.spacing(4)};
+  padding: ${({ theme }) => theme.spacing(2) + ' ' + theme.spacing(2)};
+`;
+
+const HoverSpan = styled.span`
```

Comment:
We tend to use the prefix 'Styled' to distinguish them from actual components.
See StyledTab above.

Reply:
Thanks for your feed back! Updating...

Reply:
I've made the updates by adding the 'Styled' prefix to the styled components, as you suggested. This should help improve clarity and consistency in our codebase. Please let me know if the changes meet your expectations.

Reply:
@kelvinyelyen thanks!